docs/assistant/security-assistant.asciidoc

```diff
@@ -139,5 +163,19 @@ The fields on this list are among those most likely to provide relevant context
 image::images/add-alert-context.gif[A video that shows an alert being added as context to an AI Assistant chat message]
 +
 When you include a particular event as context, you can use a similar interface to adjust anonymization behavior. Be sure the anonymization behavior meets your specifications before sending a message with the event attached.
-
++
 The *Show anonymized* toggle controls whether you see the obfuscated or plaintext versions of the fields you sent to AI Assistant. It doesn't control what gets obfuscated — that's determined by the anonymization settings. It also doesn't affect how event fields appear _before_ being sent to AI Assistant. Instead, it controls how fields that were already sent and obfuscated appear to you.
+
+* **Knowledge base:** Use retrieval-augmented generation to provide specialized knowledge of the Elastic Search Query Language ({esql}) to AI Assistant. For example, with the knowledge base active, you can ask AI Assistant to help you write an {esql} query for a particular use case, or ask it to answer general questions about {esql} syntax and usage. Without the knowledge base enabled, AI Assistant will not be able to answer questions about {esql}.
++
+beta::[]
++
+To enable the knowledge base:
++
+- Enable the Elastic Learned Sparse EncodeR (ELSER). This model provides additional context to the third-party LLM. To learn more, refer to https://www.elastic.co/guide/en/machine-learning/current/ml-nlp-elser.html#download-deploy-elser[Configure ELSER].
```

Comment:
Are these sequential steps, ie. should they be numbered?

Reply:
Small note here -- we try to do this all in one step for the user if we can. So users don't need to click `initialize` for the KB or ESQL docs unless an error has occurred or they previously deleted the ESQL docs or KB index/pipeline. Once ELSER is enabled, they will only need to flip the main KB switch and then click save:


![Kapture 2023-11-06 at 14 45 36](https://github.com/elastic/security-docs/assets/2946766/555e841d-02bc-40ff-9cb4-91414ba0cbef)